posthog/api/capture.py

```diff
@@ -96,6 +96,8 @@ def log_event(data: Dict, event_name: str, partition_key: Optional[str]):
         KAFKA_SESSION_RECORDING_EVENTS
         if event_name in SESSION_RECORDING_EVENT_NAMES
         else settings.KAFKA_EVENTS_PLUGIN_INGESTION_TOPIC
+        if partition_key
+        else settings.KAFKA_EVENTS_PLUGIN_INGESTION_OVERFLOW_TOPIC
```

Comment:
nit: I'd find it a little easier to read if we passed the kafka topic as a param to this function & then make 3 calls in `capture_internal`

Reply:
I agree with you :+1: